projects/npm-tools/packages/npm-scripts/test/utils/expandGlobs.js

```diff
@@ -220,4 +220,22 @@ describe('expandGlobs()', () => {
 			/Redundant ignore patterns/
 		);
 	});
+
+	it('can match files in a different directory', () => {
+		const randomTempDir = fs.mkdtempSync(
+			path.join(os.tmpdir(), 'scripts-basedir-')
+		);
+
+		const tempFilePath = path.join(randomTempDir, 'test.js');
+
+		fs.writeFileSync(tempFilePath, '');
+
+		const doesntMatch = expand(['*']);
+
+		expect(doesntMatch).not.toEqual([tempFilePath]);
```

Comment:
I find the double-negative here ("does not match not equal") hard to read. Might be easier as:

```js
let files = expand([*]);

expect(files).not.toInclude(tempFilePath);

files = expand(['*'], [], {baseDir: randomTempDir});

expect(files).toEqual([tempFilePath]);
```